dev/com.ibm.jbatch.container/bnd.bnd

```diff
@@ -16,7 +16,6 @@ Bundle-SymbolicName: com.ibm.jbatch.container
 Bundle-Description:  Java Batch RI core runtime implementation
 
 # java batch is part of EE7 and therefore requires java 1.7
```

Comment:
Remove this comment.

Reply:
The comment is accurate in that Java 6 is the minimum required level. If you don't want it left, I am included to remove it completely instead of updating it.

Reply:
I have removed this comment and also updated all other bnd files that had that comment.